test/crypto/rsa.js

```diff
@@ -105,6 +107,8 @@ module.exports = () => describe('basic RSA cryptography', function () {
   });
 
   it('compare native crypto and bnSign', async function() {
+    detectNative() || this.skip();
+
```

Comment:
And similarly here, I would follow the pattern from the previous test of
```suggestion
    if (!detectNative()) {
      this.skip();
    }
```
(and in the next one too)